docs/community-governance/community-governance.mdx

```diff
@@ -28,15 +28,16 @@ Network. It is also the primary venue to get support for using the Network.
 
 The Helium Community uses a governance mechanism called **Helium Improvement Proposals** (known as
 "HIPs"). Modeled on BIPs from the Bitcoin community, HIPs are used to introduce, debate, and
-approve/reject new changes to major elements of the Network like the core blockchain, $HNT token
-economics, and Network governance itself.
+approve/reject new changes to major elements of the Network like the core blockchain, $HNT, $IOT 
+$MOBILE token economics, and Network governance itself.
 
 In the earliest phases it's best to vet an idea with the Helium community
 [via Discord](https://discord.gg/helium). This is a good way to see if there is initial support for
 the idea, or if it's been discussed before.
+Start new HIP ideas in the hip-discussion channel in the Improvement Proposals folder
 
 HIPs are managed entirely from the [HIP GitHub repository](https://github.com/helium/HIP). HIP
-discussion also happens in the `Improvement Proposal` section of
+discussion also happens in the `Improvement Proposal` folder of
```

Comment:
are these not sections?

Reply:
technically "category channels" so just going with "channels"